src/components/login/login.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import LoginForm from './loginForm/LoginForm';
+import logo from '../../assets/images/logo/logo.png';
+import './login.scss';
+import { useSelector } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+
+const Login = () => {
+    const auth = useSelector((state) => state.login.isAuth);
+    if(auth) return <Redirect to='/profile'/>;
+    debugger;
```

Comment:
remove it =)

Reply:
https://github.com/mtalstykh-exadel/team4_frontend/blob/27f79d12b2ded8e5b31bd2b44a4a2d0ffd8adfa8/src/pages/login.jsx#L8-L15